helpers/error/error.go

```diff
@@ -0,0 +1,46 @@
+package errorhelpers
+
+import (
+	"net/http"
+	"os"
+	"strings"
+)
+
+func IsNotFound(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	return strings.Contains(strings.ToUpper(err.Error()), NotFound)
+}
+
+func IsTimeout(err error) bool {
```

Comment:
```suggestion
func isNotFound(err error) bool {
	if err == nil {
		return false
	}

	return strings.Contains(strings.ToUpper(err.Error()), NotFound)
}

func isTimeout(err error) bool {
```

These can be made private 🤔